drivers/soundwire/bus.c

```diff
@@ -955,33 +934,38 @@ int sdw_bus_prep_clk_stop(struct sdw_bus *bus)
 		/* Identify if Slave(s) are available on Bus */
 		is_slave = true;
 
-		slave_mode = sdw_get_clk_stop_mode(slave);
-		slave->curr_clk_stop_mode = slave_mode;
```

Comment:
Should we remove `curr_clk_stop_mode` from `struct sdw_slave {}`? It is not used anymore.

Reply:
yes we could. we can also remove the use of MODE1 that I kept

Reply:
done